ui/media/css/main.css

```diff
@@ -250,6 +250,8 @@ button#resume {
 }
 img {
     box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.15), 0 6px 20px 0 rgba(0, 0, 0, 0.15);
+    width:  100%;
```

Comment:
No, the 100% is relative to the image's size, not to the container's available space.
The "object-fit:contain" seems to impact the "Buy me a coffee" image so that it becomes centered.
Change it to `div.img-preview img { width:100%; object-fit:contain;}`, so that it only affects preview images?

Reply:
Thanks guys! Actually the right fix is:
div.img-preview img { width:100%; height: 100%;}

![image](https://user-images.githubusercontent.com/48073125/211367572-26d4a8af-601b-4aca-8265-2f3c467b9e4d.png)
![image](https://user-images.githubusercontent.com/48073125/211367796-83f635f9-e781-4a19-800a-04baec6bccf0.png)

Reply:
And no, it doesn't cause the entire image to take 100% of the page. It just causes it to fit its own container. I also confirm the Buy me a coffee button renders as expected with JeLuf's proposal.

![image](https://user-images.githubusercontent.com/48073125/211368295-5da54639-0c48-44d6-acc1-b95f4bcb3e9f.png)